.github/workflows/lint.yaml

```diff
@@ -64,6 +64,6 @@ jobs:
         uses: gsactions/commit-message-checker@16fa2d5de096ae0d35626443bcd24f1e756cafee
         with:
           accessToken: ${{ secrets.GITHUB_TOKEN }}
-          pattern: '(b\/\d+|^(Bug|Fixed|Issue): \d+$)'
+          pattern: '(b\/\d+|^(Bug|Fixed|Issue): \d+$|partnerissuetracker\.corp\.google\.com\/u\/\d+\/issues/\d+$)'
```

Comment:
Lets make sure `issuetracker.google.com` is also accepted here. That's where we link to from cobalt.dev comms guide as well.

Reply:
Done

Reply:
Actually still needs a small revision : external link is http://issuetracker.google.com, so you probably want (issuetracker|partnerissuetracker) here.

Our public issuetracker link is: https://issuetracker.google.com/issues?q=componentid:181120%20status:open

Reply:
Changed that pattern to `(partnerissuetracker\.corp|issuetracker)`. I've verified all original bug patterns and the internal and external issue trackers links manually.